spec/models/item_spec.rb

```diff
@@ -1,7 +1,12 @@
 require 'rails_helper'
 
 RSpec.describe Item, type: :model do
-  let(:item) {Fabricate(:item)}
+  let(:item) { item = Fabricate.build(:item)
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.